index.android.js

```diff
@@ -9,45 +9,186 @@ import {
+        </View>
       </View>
     );
   }
 }
 
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    justifyContent: 'center',
-    alignItems: 'center',
-    backgroundColor: '#F5FCFF',
+function toCelsius(k) {
+  return k - 273.15
+}
+
+const iconNames = {
+	Default: 'md-time',
+	Clear: 'md-sunny',
+	Rain: 'md-rainy',
+	Thunderstorm: 'md-thunderstorm' ,
+	Clouds: 'md-cloudy' ,
+	Snow: 'md-snow' ,
+	Drizzle: 'md-umbrella' ,
+	Mist: 'md-partly-sunny'
+}
+
+const phrases = {
+	Default: {
+		title: "Fetching the Weather",
+		subtitle: "Be patient, you're witnessing a miracle",
+		highlight: "miracle",
+		color: "#636363",
+		background: "#9C9C9C"
+	},
+	Clear: {
+		title: "It's Amazing",
+		subtitle: "Rock that shit!",
+		highlight: "Enjoy",
+		color: "#E32500",
+		background: "#FFD017"
+	},
+	Rain: {
+		title: "Rain rain please go away",
+		subtitle: "Stay inside and code all day",
+		highlight: "away",
+		color: "#004A96",
+		background: "#2F343A"
+	},
+	Thunderstorm: {
+		title: "Watch out, Thunder Strike",
+		subtitle: "Unplug those devices",
+		highlight: "Thunder",
+		color: "#FBFF46",
+		background: "#020202"
+	},
+	Clouds: {
+		title: "Cloud storage limit reached",
+		subtitle: "error: 5000 - cirrocumulus",
+		highlight: "limit",
+		color: "#0044FF",
+		background: "#939393"
+	},
+	Snow: {
+		title: "of course it's snowing",
+		subtitle: "You're not supposed to eat it",
+		highlight: "Snow",
+		color: "#021D4C",
+		background: "#15A678"
+	},
+	Drizzle: {
+		title: "Meh... don't even ask",
+		subtitle: "What did I just say?",
+		highlight: "don't",
+		color: "#B3F6E4",
+		background: "#1FBB68"
+	},
+	Mist: {
+		title: "Mist",
+		subtitle: "Mist !",
+		highlight: "Mist",
+		color: "#B3F6E4",
+		background: "#1FBB68"
+	},
+}
+
+const allStyles = StyleSheet.create({
+  container:{
+    flex:1,
+    backgroundColor:`#FFD017`
   },
-  welcome: {
-    fontSize: 20,
-    textAlign: 'center',
-    margin: 10,
+  iconStyle:{
+  fontSize: 90,
+  color: 'white',
+  marginBottom: -80
+
+  },
+  headerStyle:{
+    alignItems:'center',
+    justifyContent:'space-around',
+    flex:1,
+   // backgroundColor: 'red',
+    flexDirection:'row'
+  },
+  temp:{
+    fontFamily:'HelveticaNeue-Bold',
+    fontSize:55,
+    color:'white',
+    marginBottom: -80
   },
-  instructions: {
-    textAlign: 'center',
-    color: '#333333',
-    marginBottom: 5,
+  bodyStyle:{
+    alignItems:'flex-start',
+    justifyContent:'flex-end',
+    flex:5,
+    margin:10,
+   // backgroundColor: 'yellow'
   },
-});
+  title:{
+    fontFamily:'HelveticaNeue-Bold',
+    alignItems:'center',
+    fontSize:55,
+    color:'white',
+    marginBottom:200,
+  },
+  subTitle:{
+    fontFamily:'HelveticaNeue-Medium',
+    fontSize:24,
+    color:'white',
+    marginBottom:-444
+  }
+})
 
-AppRegistry.registerComponent('SimpleWeatherApp', () => SimpleWeatherApp);
+AppRegistry.registerComponent('SimpleWeatherApp', () => App);
```

Comment:
Why did you change the name of the App to App ? it should be SimpleWeatherApp

Reply:
The app name is determined by what's between the '' marks. in this case it is 'SimpleWeatherApp'. The word App in this case is the class name.

Reply:
👍